docs/create_tasks_table.py

```diff
@@ -56,15 +65,49 @@ def create_tasks_table(tasks: list[mteb.AbsTask]) -> str:
     return table
 
 
-def insert_table(file_path, table):
-    """Insert table in the in <!-- TABLE START --> and <!-- TABLE END -->"""
+def create_task_lang_table(tasks: list[mteb.AbsTask]) -> str:
+    
+    table_dict = {}
+    ## Group by language. If it is a multilingual dataset, 1 is added to all languages present.
+    ## For now, we skip cross lingual datasets.
```

Comment:
Was discussing this with @imenelydiaker: A cross-lingual dataset would have 2 languages, but it's still 1 dataset. However, we count each language within a multilingual dataset. Wanted some feedback on this.

Reply:
I would add 1 for each language in a cross-lingual dataset as well.

Reply:
otherwise all bitext becomes zero which seems odd.

Reply:
Adding back cross lingual datasets.